keps/sig-scheduling/902-non-preempting-priorityclass/README.md

```diff
@@ -199,18 +205,117 @@ Ensure existing tests (for preempting PriorityClasses) do not break.
 
 ### Rollout, Upgrade and Rollback Planning
 * **How can a rollout fail? Can it impact already running workloads?**
-  The scheduler errors and exits during start up. Existing workloads are not
-  affected.
+  If a rollout fails, kube-scheduler will keep crashing. Running workloads won't be affected by kube-scheduler.
 
 * **What specific metrics should inform a rollback?**
-  N/A.
+Check the following indicators to determine if there are any exceptions:
+  - pod_preemption_victims
+  - total_preemption_attempts
+  - scheduling_algorithm_preemption_evaluation_seconds
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
-  N/A.
+  Manually tested successfully.
 
 * **Is the rollout accompanied by any deprecations and/or removals of features?**
   N/A.
 
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+N/A. This is not a feature that workloads use directly
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
+
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [ ] Events
```

Comment:
Does the scheduling event include the detail of whether pods were preempted. Is there anything we can improve?

Reply:
Yes，there is a scheduling event include the detail of pods were preempted by other. Added.